api/http/api_inventory.go

```diff
@@ -643,3 +716,24 @@ func (i *inventoryHandlers) InternalDevicesStatusHandler(w rest.ResponseWriter,
 
 	w.WriteHeader(http.StatusOK)
 }
+
+func parseSearchParams(r *rest.Request) (*model.SearchParams, error) {
+	var searchParams model.SearchParams
+
+	if err := r.DecodeJsonPayload(&searchParams); err != nil {
+		return nil, errors.Wrap(err, "failed to decode request body")
```

Comment:
adding this unit test scenario will make codecov happy.

Reply:
done